app/services/glassdoor_service.rb

```diff
@@ -12,9 +12,16 @@ def company(company_name)
   private
 
   def parse(response)
-    results = JSON.parse(response.body)["response"]
+    results = exact_matches(JSON.parse(response.body)["response"])
+
     if results
-      results["employers"].first
+      results.first
```

Comment:
I guess I don't understand the change, isn't the way it was pulling only the first result?